Packs/FeedElasticsearch/Integrations/FeedElasticsearch/FeedElasticsearch.py

```diff
@@ -208,36 +209,53 @@ def get_demisto_indicators(search, tags, tlp_color):
     return ioc_lst, ioc_enrch_lst
 
 
-def fetch_indicators_command(client, feed_type, src_val, src_type, default_type, last_fetch):
+def update_last_fetch(client, ioc_lst):
+    last_calculated_time = None
+    last_ids = []
+    for ioc in reversed(ioc_lst):
+        calculate_time = dateparser.parse(ioc.get(client.time_field))
+        if calculate_time and (not last_calculated_time or calculate_time > last_calculated_time):
+            last_calculated_time = calculate_time
+            last_ids.append(ioc.get('id'))
+        else:
+            break
+    if last_calculated_time is None:
+        last_calculated_time = datetime.now()
```

Comment:
if last calculated time is None, meaning probably we did not fetch any indicators, shodden we just preserve the preserve time we fetched with on the current fetch?

Reply:
We want to update the last run to now so the query will be as efficient as possible.
If we didn't receive any indicators, we can update it to the `now` time.